blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseBlobIntegrationTest.java

```diff
@@ -432,7 +433,13 @@ public void deleteObject(String name) throws InterruptedException {
       String container = getContainerName();
       try {
          addBlobToContainer(container, name, name, MediaType.TEXT_PLAIN);
+         if (view.getConsistencyModel() == ConsistencyModel.EVENTUAL) {
```

Comment:
Well but this just applies a wait only the case of eventual consistency providers. How will the assert work here?